specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,689 @@
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Managed CCF Create Request Body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was created; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          },
+          "201": {
+            "description": "The Managed CCF resource will be created asynchronously"
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedCCFCreate": {
+            "$ref": "./examples/ManagedCCF_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Update Managed CCF properties",
+        "description": "Updates properties of Managed CCF",
+        "operationId": "ManagedCCF_Update",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Request body for Updating Managed CCF App",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and Managed CCF was recovered; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          },
+          "201": {
+            "description": "The Managed CCF resource will be updated asynchronously"
```

Comment:
>description [](http://example.com/codeflow?start=13&length=11)

this doesn't make much sense. If its an asynchronous update the response code should be 202. If its an asyncronous create ,the recommended pattern is return Created, with a provisioingState .

Reply:
"patch" verb isn't recommended to be able to create though. Its supposed to be updates only, in RPC guidelines.

Reply:
Reverted back to 202 which was what it was for the previous preview version. I accidentally had changed it.